figures/views.py

```diff
@@ -263,7 +263,10 @@ class GeneralCourseDataViewSet(CommonAuthMixin, viewsets.ReadOnlyModelViewSet):
 
     '''
     model = CourseOverview
-    pagination_class = FiguresLimitOffsetPagination
+
+    # The "kilo paginator"  is a tempoarary hack to return all course to not
+    # have to change the front end until Figures "Level 2"
+    pagination_class = FiguresKiloPagination
```

Comment:
FYI, `FiguresKiloPagination` is not imported in `views.py`, so this should be a runtime error. Flake8 correctly points it out, which is why the TravisCI tests are failing.

Reply:
@thraxil Thank you very much! Fixed an added test